src/views/Player.vue

```diff
@@ -264,19 +244,10 @@ export default {
       }
 
       this.updateJourney(
-        "questionJourney", "submit", String("q-" + currQuesIndex)
-      )
-
-      // commented these right now,
-      // if plan changes in future
-
-      // this.questionJourney.push(
-      //   [
-      //     String(this.getCurrentDateTime()),
-      //     "submit",
-      //     String("q-" + currQuesIndex)
-      //   ]
-      // )
+          "question-submitted", {
+            'question': currQuesIndex,
+            'option': plioQuestion.item.question.options.indexOf(answer) + 1
```

Comment:
for submit response, log the question number and option selected as details.

Reply:
indexOf might not work in older browsers!

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/indexOf#Polyfill